docs/upgrading/1.5-to-1.6.md

```diff
@@ -1,19 +1,24 @@
-## Version 1.6
-
-### eosiolib
-#### Partitioning
-In `eosio.cdt` v1.6.0, `eosiolib` will now be partitioned into 3 groups. These allow for finer grained allowance for particular modes of compilation.
-  - CAPI
-  - Contracts
-  - Core
-  - The `CAPI` will only be available to contract writers using `eosio-cc` and purely for `C` smart contracts.
-  - The `Contracts` will be available to `eosio-cpp` for smart contract writing. 
-  - `Core` will be available to `eosio-cpp` for any of the modes (present and future).  
-  - `eosio-cpp` in `fnative` mode will have access to all of these groups.
-
-To access these new partitioned header files, use `<eosio/header_file.hpp>` instead of `<eosiolib/header_file.hpp>`.  Please note that all the old header files are still available at the old `eosiolib` directory, but these are deprecated and will be removed in v1.7.0.  Also, once you change one header file from `eosiolib` to `eosio` you will need to do so for all other occurrences at that point, because of some conflicts with the auto generated dispatcher.
-
-### eosiolib C API
+ \defgroup v15-to-v16 1.5 To 1.6
```

Comment:
It's no migration from 1.5 to 1.6, due we public 2.0.

Reply:
the upgrading directory has been removed from description.